packages/catalog-process/src/services/catalogService.ts

```diff
@@ -210,7 +222,7 @@ const hasNotDraftDescriptor = (eService: EService): void => {
   }
 };
 
-const updateDescriptor = (
+const replaceDescriptor = (
```

Comment:
Renaming. This is just a util function that replaces the descriptor with its updated version (e.g. a state change). This should not be confused with the business logic function `updateDescriptor`.